modules/authorization/policy-assignments/subscription/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Policy Assignment on Subscription level",
+  "summary": "With this module you can perform policy assignments on a subscription level.",
```

Comment:
Should the README.md be updated to "This module deploys Policy Assignments on a Subscription level." and then this file updated accordingly?